book/src/requirements.md

```diff
@@ -0,0 +1,102 @@
+
+Finally, there is a category of proposed requirements that could have been coherently in-scope for Orchard which were excluded due to time & complexity constraints.
+
+These scope clarifications exlude some draft requirements found in earlier revisions of this document, in pull request comments, or elsewhere. Many of those could become ZIPs in their own right.
+
+## Protocol Product Requirements
+
+### ProtoPR1. Orchard Relies on a Zeroknowledge Proving System which is Secure in The Uniform Random String Model
+
+**Requirement:** The Orchard shielded transfer circuit uses a Zeroknowledge Proving System which is secure in the Uniform Random String (URS) Model, and therefore has no reliance on a trusted setup.
+
+**Rationale - UX:** Users who rely on Orchard are not directly vulnerable to the class of trusted parameter compromises.
+
+**Rationale - Strategic:** As Sprout and Sapling usage declines or are deprecated, the ZEC a whole will become protected from any previous trusted setup compromise henceforth.
+
+**Rationale - Strategic:** By avoiding a requirement on a trusted setup, Orchard *and future* upgrades no longer need a parameter setup MPC process, which improves protocol agility.
+
+### ProtoPR2. Orchard Addresses
+
+**Requirement:** Orchard introduces a new address format with an unambiguous user-facing encoding.
+
+**Rationale - UX:** A new address format allows users to identify which ZKP platform their funds rely on.
+
+**Rationale - UX:** A new address format reinforces the fact that users must migrate funds to opt into Orchard.
+
+**Rationale - UX:** A new address format, in combination with the turnstile design, signals to users a boundary for turnstile-based supply integrity protections.
+
+**Rationale - UX:** A new address format, in combination with the turnstile design, signals to users a boundary for privacy-guarantees.
+
+**Counter-rationale - UX:** This complicates the simplified narrative of "z→z has strong privacy" because migrating between pools has a privacy cost.
+
+**Rationale - Engineering:** A new address format simplifies cryptographic design and performance.
+
+### ProtoPR3. An Orchard Shielded Pool and Turnstile
+
+**Requirement:** A new shielded pool is introduced with Orchard. ZEC may only enter the Orchard pool from t-addrs or shielded migration transactions, following the same "turnstile" design as for Sprout and Sapling.
+
+**Rationale - UX:** A separate shielded pool clarifies which funds rely on which underlying ZKP/cryptographic platform.
+
+**Rationale - UX:** A separate shielded pool associated with a new address type makes it easier to reason about migration.
+
+### ProtoPR4. Orchard Privacy Protections are Independent of Soundness.
+
+**Requirement:** Orchard's privacy protections are independent of proving system soundness so that a soundness compromise does not compromise any privacy protections.
```

Comment:
```suggestion
**Requirement:** Orchard's privacy protections are independent of proving system soundness (in the sense that the security arguments are disentangled from each other and can be separately verified) so that a soundness compromise does not compromise any privacy protections.
```